x-pack/plugins/cases/public/components/case_action_bar/index.test.tsx

```diff
@@ -249,4 +223,38 @@ describe('CaseActionBar', () => {
     userEvent.click(screen.getByTestId('property-actions-ellipses'));
     expect(queryByText('Delete case')).toBeInTheDocument();
   });
+
+  it('shows the external incident action', async () => {
+    const connector = caseConnectors['servicenow-1'];
+    const { push, ...connectorWithoutPush } = connector;
+
+    const props = {
+      ...defaultProps,
+      caseData: { ...defaultProps.caseData, connector: connectorWithoutPush },
+    };
+
+    render(
+      <TestProviders>
+        <CaseActionBar {...props} />
+      </TestProviders>
+    );
+
+    userEvent.click(screen.getByTestId('property-actions-ellipses'));
+
+    await waitFor(() => {
+      expect(screen.getByTestId('property-actions-popout')).toBeInTheDocument();
+    });
+  });
+
+  it('does not show the external incident action', async () => {
+    render(
+      <TestProviders>
+        <CaseActionBar {...defaultProps} />
+      </TestProviders>
+    );
+
+    userEvent.click(screen.getByTestId('property-actions-ellipses'));
+
+    expect(screen.queryByTestId('property-actions-popout')).not.toBeInTheDocument();
```

Comment:
This reminds me of something. It is a minor thing but in this popover

<img width="207" alt="Screenshot 2023-01-26 at 11 20 15" src="https://user-images.githubusercontent.com/1533137/214812282-2e6d8946-8881-4f06-87ec-16071dd4db27.png">

I feel like the delete/red option would look better as the last.

Reply:
You are right! I will fix it.